src/apps/wallet/ecdh.py

```diff
@@ -4,7 +4,8 @@
 from ustruct import pack, unpack
 from trezor.utils import chunks
 from apps.common.confirm import require_confirm
-from apps.wallet.sign_identity import serialize_identity
+from apps.wallet.sign_identity import (serialize_identity,
```

Comment:
Style nitpick: please don't use braces and keep import on one line.

Reply:
Done.